test/e2e/network/netpol/probe.go

```diff
@@ -90,27 +88,22 @@ func probeWorker(k8s *kubeManager, jobs <-chan *ProbeJob, results chan<- *ProbeJ
 	defer ginkgo.GinkgoRecover()
 	for job := range jobs {
 		podFrom := job.PodFrom
-		containerFrom, err := podFrom.FindContainer(int32(job.FromPort), job.Protocol)
-		// 1) sanity check that the pod container is found before we run the real test.
-		if err != nil {
-			result := &ProbeJobResults{
-				Job:         job,
-				IsConnected: false,
-				Err:         err,
-				Command:     "(skipped, pod unavailable)",
-			}
-			results <- result
-		} else {
-			// 2) real test runs here...
-			connected, command, err := k8s.probeConnectivity(podFrom.Namespace, podFrom.Name, containerFrom.Name(), job.PodTo.QualifiedServiceAddress(job.ToPodDNSDomain), job.Protocol, job.ToPort)
-			result := &ProbeJobResults{
-				Job:         job,
-				IsConnected: connected,
-				Err:         err,
-				Command:     command,
-			}
-			results <- result
+
+		connected, command, err := k8s.probeConnectivity(
```

Comment:
not sure if this was meant to be reformatted or not, I think generally it's done in one line in kube !